Vagrantfile.example

```diff
@@ -39,6 +39,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provision "ansible" do |ansible|
       # ansible.verbose = "vvv"
       ansible.playbook = "ansible/pulp-from-source.yml"
+      # ansible.start_at_task = "ansible task name here"
+      # ansible.extra_vars = { pulp_venv: "/home/vagrant/.virtualenvs/pulp" }
```

Comment:
Did you mean to commit these commented out lines?

Reply:
I did, I think they will be useful settings again. Really L#42 is the useful one, but then you also need L#43.